src/plugins/controlbox/tests/login.js

```diff
@@ -3,6 +3,25 @@
 const u = converse.env.utils;
 
 describe("The Login Form", function () {
+    
+    it("contains an addon when locked_domain is set", 
```

Comment:
```suggestion
    it("shows the domain when locked_domain is set", 
```

The word `addon` is very ambiguous.